test/lib-scripts.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const {
+    parseDataAttributes,
+    parseScripts,
+} = require('../lib/scripts');
+
+test('it does not mutate the original script', () => {
```

Comment:
You could put another test here if you want to make sure it doesn't needlessly duplicate the object, kinda worthless, but would be interesting to validate my first comment.